src/traces/image/plot.js

```diff
@@ -181,13 +174,21 @@ module.exports = function plot(gd, plotinfo, cdimage, imageLayer) {
             }
         })
         .then(function() {
-            var href, canvas;
+            var href, canvas, localStyle;
+            localStyle = initialStyle;
             if(trace._hasZ) {
                 canvas = drawMagnifiedPixelsOnCanvas(function(i, j) {return z[j][i];});
                 href = canvas.toDataURL('image/png');
             } else if(trace._hasSource) {
                 if(fastImage) {
                     href = trace.source;
+                    // Flip the SVG image as needed (around the proper center location)
+                    var axisScale = [(xa.range[0] < xa.range[1]) ? 1 : -1, (ya.range[0] < ya.range[1]) ? -1 : 1];
+                    var trans = '';
+                    trans += 'translate(' + (left + imageWidth / 2) + 'px,' + (top + imageHeight / 2) + 'px)';
```

Comment:
@almarklein I think I tried to use `transofrm-origin` in the past but it's one of the things that behave differently in SVG than in HTML. It looks like chaining the transforms as you did here is the way to go in SVG. :1st_place_medal: 

> Just like HTML elements, SVG elements can be manipulated using transform functions. However, a lot of things don’t work the same way on SVG elements as they do on HTML elements.

https://css-tricks.com/transforms-on-svg-elements/

Reply:
Yeah, and then there is the fact that the behavior of `transform-origin` for SVG is different for Firefox than it is for other browsers :/  I've read that the Firefox-way (relative to the svg canvas) is the correct one. Unfortunately, that approach is also useless for our use-case.